fec_eregs/static/fec_eregs/js/data/terms.json

```diff
@@ -103,6 +103,10 @@
     "term": "Contribution in the name of another",
     "definition": "Giving money or anything of value, all or part of which was provided to the contributor by another person (the true contributor) without disclosing the source of the money or the thing of value to the recipient candidate or committee at the time the contribution is made; or making a contribution of money or anything of value and attributing as the source another person when in fact the contributor is the source. <a href=\"https://www.fec.gov/regulations/110-4/CURRENT#110-4-b\">11 CFR 110.4(b)</a>."
   },
+  {
+    "term": "Cooperatives",
+    "definition": "A cooperative is a voluntary economic association organized to provide economic services, without gain, to itself, and to the members who own and control it. A federated cooperative as defined in the Agricultural Marketing Act of 1929, 12 U.S.C. 1141j, or a rural cooperative eligible for assistance under chapter 31 or title 7 of the United States Code, may solicit the members of the cooperative's regional, state or local affiliates, provided that all of the political committees established, financed, maintained or controlled by the cooperative and its regional, State or local affiliates are considered one political committee for the purposes of the limitations in 11 CFR <a href=\"https://www.fec.gov/regulations/110-1/CURRENT#110-1\">110.1</a> and <a href=\"https://www.fec.gov/regulations/110-2/CURRENT#110-2\">110.2</a>."
+  },
```

Comment:
Need to make State lower case s.

Reply:
Actually a question too - should it be Cooperative (singular) or Cooperatives (plural)? Looking at other glossary terms, I think it should be singular with no s on the end unless there's a design reason that @rfultz and @JonellaCulmer want it to be plural.

Reply:
Changes made

Reply:
I'd like to take the s out of cooperative too, I think.

Reply:
S removed from cooperative